src/foam/u2/wizard/wao/SplitWAO.js

```diff
@@ -51,10 +51,14 @@ foam.CLASS({
     },
     async function save (wizardlet) {
       const saver = foam.json.parse(this.saver, undefined, wizardlet.__subContext__);
-      if ( wizardlet.loading ) return;
-      if ( ! wizardlet.isAvailable ) return;
-      wizardlet.loading = true;
-      await saver.save(wizardlet.data);
+      if ( this.NullSaver.isInstance(saver) ) {
```

Comment:
This seems odd. The idea of having a Null strategy is to make it transparent when you aren't saving. Having to do this to save anyway seems to defeat the purpose.